index.js

```diff
@@ -1,168 +1,130 @@
-// let secondNumber = +prompt('введіть друге число');
-// let add = `${firstNumber}+${secondNumber} = ${firstNumber + secondNumber}`;
-// let sub = `${firstNumber}-${secondNumber} = ${firstNumber - secondNumber}`;
-// let mult = `${firstNumber}*${secondNumber} = ${firstNumber * secondNumber}`;
-// let div = `${firstNumber}/${secondNumber} = ${firstNumber / secondNumber}`;
-// let resolt = `Ваш результат:\n${add}\n${sub}\n${mult}\n${div}`;
-// alert(resolt);
-
-
-//homework2/1
-
-// let surName = prompt ('Enter your last name');
-
-// let firstName = prompt('Enter your firs name');
-
-// let patronymic = prompt('Enter your middle name');
-
-// let greeting = 'good morning ' + surName + ' ' + firstName + ' ' + patronymic;
-// // let greeting = `Good Morning ${surName} ${firstName} ${patronymic}`;
-
-
-// alert(greeting);
-
-
-
-
-// HomeWork2/2
-
-// let number =  92358;
-// let string = String(number);
-// let newString = '';
-// for(i = 0; i < string.length ; i++){
-//     newString += `${string[i]} `;
-// };
-// console.log(newString);
-=======
-let firstNumber = +prompt('введіть перше число');
-let secondNumber = +prompt('введіть друге число');
-let thirdNumber = +prompt('введіть третє число');
-let arithmeticMean = `${(firstNumber+secondNumber+thirdNumber)/3}`;
-alert(arithmeticMean);
-
-const SECONDS_IN_HOUR = 3600;
-let hours = +prompt('введіть кількість годин які хочете перевести в секунди');
-let seconds = `${hours * SECONDS_IN_HOUR} секунд`;
-alert(seconds);
-
-let condition = false;
-let getUserLengthArray;
-do {
-    getUserLengthArray = prompt("Введіть довжину масиву");
-    if (isNaN(getUserLengthArray) === true) {
-        //return "Please imput a number.";
-        alert("Будь ласка, введіть число");
-        condition = false;
-    } else {
-        condition = true;
+class House {
+    apartmentsArray
+    constructor(params) { 
+        this.city = params.city
+        this.street = params.street
+        this.houseNumber = params.houseNumber
+        this.numberOfApartments = params.numberOfApartments
+        this.apartmentsArray = [];
     }
-} while (condition === false);
-
-const newArray = [];
-for (let i = 0; i < getUserLengthArray; i++) {
-    newArray[i] = prompt(`Введіть елемент масиву № ${i + 1}`);
-    if (newArray[i] === "" || newArray[i] === null) {
-        delete newArray[i];
-    } else if (!isNaN(+newArray[i])) {
-        newArray[i] = Number(newArray[i]);
+    apartmentNumber = 0;
+    addApartment( areaOfApartment, numberOfRooms, apartmentNumber = this.apartmentNumber+1){
+        if(this.apartmentNumber < this.numberOfApartments){
+            this.apartmentsArray.push(new Apartment(areaOfApartment, numberOfRooms, apartmentNumber))
+            this.apartmentNumber++
+        } return this.apartmentsArray
     }
-}
-
-console.log(newArray);
-console.log(newArray.sort().sort((a, b) => a - b));
-newArray.splice(1, 3);
-console.log(newArray);
-
-
-
+    getHouseInfo(){
```

Comment:
```suggestion
   logHouseInfo(){
```

Ви ж не повертаєте інформацію, а виводите її у консоль